staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/admission.go

```diff
@@ -175,7 +182,7 @@ func (c *celAdmissionPlugin) Validate(
 		return admission.NewForbidden(a, fmt.Errorf("not yet ready to handle request"))
 	}
 
-	return c.evaluator.Validate(ctx, a, o)
+	return c.evaluator.Validate(ctx, a, o, c.authorizer)
```

Comment:
the authorizer is the same for every request, should that be baked into the evaluator in `NewAdmissionController` rather than passed to Validate?

Reply:
Oops, fixed.

Reply:
huh... now that c.evaluator has the authorizer, I didn't expect us to need to also plumb it to the `Validate` call as a parameter

Reply:
Apologies, I missed this in the last edit. Fixed.